draft-ietf-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -842,6 +842,19 @@ ULP:  Upper Layer Protocol
    user messages that are buffered in the SCTP layer MUST be read and
    processed by DTLS.
 
+   The interaction between peers and protocol stacks shall be as follows:
+1. Local peer A asks for terminating the DTLS/SCTP Association
+2. Local DTLS/SCTP acknowledge the request, from this time on no new messages will be accepted from A
+3. Local DTLS/SCTP flushes all data towards the remote peer B
```

Comment:
Instead of flushes I would use something that makes it clears that DTLS/SCTP finish any protection operation on buffered user messages and ensures that all protected user message data has been accepted by SCTP for transmission, i.e. it needs to be in the tx_buffer.